js/WebView.android.js

```diff
@@ -41,10 +49,52 @@ const defaultRenderLoading = () => (
   </View>
 );
 
+type State = {|
+  viewState: WebViewState,
+  lastErrorEvent: ?WebViewErrorEvent,
+  startInLoadingState: boolean,
+|};
+
+type WebViewPropsAndroid = $ReadOnly<{|
+  ...WebViewSharedProps,
+  onNavigationStateChange?: (event: WebViewEvent) => any,
```

Comment:
The return type of the callback should be `mixed` instead of `any`: https://flow.org/en/docs/types/functions/#toc-function-type